### DISCUSSION TITLE
Clear queue or resume last track

### DISCUSSION THREAD

**Discussion by erkr** [music-assistant/support#726] [Feature requests and ideas]
When either power cycling a player or restarting HA/MA, the current MA remembers the queue per player.
 
Now I wonder why these queues are preserved in the first place. Especially because playback will start again with the first track, not where it was stopped the last run.

Personally I don't like the current behavior, but I could perfectly live with simply clearing these queue's.
Another great improvement would be to remember the last playback position.

**Comment by marcelveldt** [comment]
It will resume wherever it stopped so this is either a bug or a result of your "hard killing" of players.

**Comment by OzGav** [comment]
Relates to v1
